src/algebra/lie/cartan_matrix.lean

```diff
@@ -147,7 +168,98 @@ end cartan_matrix
 
 Note that it is defined for any matrix of integers. Its value for non-Cartan matrices should be
 regarded as junk. -/
-@[derive [lie_ring, lie_algebra R]]
+@[derive [inhabited, lie_ring, lie_algebra R]]
 def matrix.to_lie_algebra := (cartan_matrix.relations.to_ideal R A).quotient
 
-instance (A : matrix B B ℤ) : inhabited (matrix.to_lie_algebra R A) := ⟨0⟩
+namespace cartan_matrix
+
+/-- The Cartan matrix of type e₆. See `bourbaki1968` plate V, page 277.
+
+The corresponding Dynkin diagram is:
+            o
+            |
+o --- o --- o --- o --- o
+-/
+def E₆ : matrix (fin 6) (fin 6) ℤ := ![![ 2,  0, -1,  0,  0,  0],
+                                       ![ 0,  2,  0, -1,  0,  0],
+                                       ![-1,  0,  2, -1,  0,  0],
+                                       ![ 0, -1, -1,  2, -1,  0],
+                                       ![ 0,  0,  0, -1,  2, -1],
+                                       ![ 0,  0,  0,  0, -1,  2]]
+
+/-- The Cartan matrix of type e₇. See `bourbaki1968` plate VI, page 281.
```

Comment:
Are you sure this shouldn't be 
```suggestion
/-- The Cartan matrix of type e₇. See [bourbaki1968] plate VI, page 281.
```

Reply:
Oh cool, I thought this feature was still WIP but it seems to work.